build.py

```diff
@@ -68,23 +31,20 @@ def get_local_path(package_name, path_elements):
     log.debug('local path: {0}'.format(local_path))
     return local_path
 
-def run_cmd(cmd, run_env=False, unsafe_shell=False, check_rc=False, retries=0):
+def run_cmd(cmd, run_env=None, unsafe_shell=False, check_rc=False, retries=0):
     log = logging.getLogger(__name__)
-    # run it
-    if run_env == True:
-        set_rvm_path()
-
-    run_env = os.environ.copy()
+    if run_env is not None:
+        run_env = os.environ.copy()
```

Comment:
The `run_env` parameter and its usage isn't clear to me. Is L36 supposed to be `is None` instead of `is not None`?

Reply:
Yes, nice catch

Reply:
Fixed